Description: Remove builtin which causes 32-bit ARM builds to fail
Author: vector-of-bool <vectorofbool@gmail.com>
Forwarded: not-needed
diff --git a/src/mc-range-edge-generation-private.h b/src/mc-range-edge-generation-private.h
index 7e1a0192..63718963 100644
--- a/src/mc-range-edge-generation-private.h
+++ b/src/mc-range-edge-generation-private.h
@@ -93,18 +93,6 @@ mc_getEdgesDecimal128 (mc_getEdgesDecimal128_args_t args,
 static inline size_t
 mc_count_leading_zeros_u64 (uint64_t in)
 {
-#ifdef __has_builtin
-#if __has_builtin(__builtin_clzl)
-// Pointer-cast to ensure we are speaking the right type
-#if defined(__APPLE__) || defined(__ILP32__)
-   unsigned long long *p = &in;
-   return (size_t) (in ? __builtin_clzll (*p) : 64);
-#else
-   unsigned long *p = &in;
-   return (size_t) (in ? __builtin_clzl (*p) : 64);
-#endif
-#endif
-#endif
    uint64_t bit = UINT64_C (1) << 63;
    size_t count = 0;
    while ((bit & in) == 0 && bit > 0) {
